.github/workflows/kind.yaml

```diff
@@ -65,9 +71,19 @@ jobs:
             --helm-set tls.secretsBackend=k8s
 
       - name: Enable Relay
+        if: ${{ matrix.mode == 'classic' }}
         run: |
           cilium hubble enable --ui
 
+      # TODO: Implement 'cilium hubble enable' command
```

Comment:
How difficult is it to do this?

Reply:
extremely easy. just waiting for this PR to get merged before marking https://github.com/cilium/cilium-cli/pull/1479 ready for review 😊